system_files/just/custom.just

```diff
@@ -10,6 +10,9 @@ framework-13:
     rpm-ostree kargs --append="module_blacklist=hid_sensor_hub"
   fi
 
+framework-13-amd:
+  rpm-ostree kargs --append="amdgpu.sg_display=0" --append="rtc_cmos.use_acpi_alarm=1"
```

Comment:
> amdgpu.sg_display=0

This is a workaround that has some pretty dramatic side effects, it should only be used by people that need it rather than everyone.

> rtc_cmos.use_acpi_alarm=1

IMO it's cleaner to carry the kernel patch in the kernel.
https://patchwork-proxy.ozlabs.org/project/rtc-linux/patch/20231106162310.85711-1-mario.limonciello@amd.com/

Reply:
Thanks for the review Mario! Ok I'll just no-op the command for now. We're not set up to carry patches but if someone COPRs it we could.

Reply:
Can someone request the Fedora kernel to start carrying that patch maybe?

Reply:
> Can someone request the Fedora kernel to start carrying that patch maybe?

I'll reach out to them today. Thanks Mario.

Reply:
> > amdgpu.sg_display=0
> 
> This is a workaround that has some pretty dramatic side effects, it should only be used by people that need it rather than everyone.

This is a great point. @castrojo, would it make sense to have a "just blah" for those who experience the need for it? In the docs, I mentioned it under optional as ideally, we would not want everyone using it. It's an e-brake of sorts. :)

Reply:
Update. Just updated to 6.5.11, not seeing the behavior with suspend/wake we were using with rtc_cmos.use_acpi_alarm=1 right now. Additionally, [this script when run](https://gitlab.freedesktop.org/drm/amd/-/blob/master/scripts/amd_s2idle.py) is no longer calling out [what we were seeing here](https://community.frame.work/t/responded-high-gpu-power-draw-on-amd-ryzen-7-7840u/39713/17?u=matt_hartley). @castrojo maybe hold off on rtc_cmos.use_acpi_alarm=1 for now - I need to dig into this a bit deeper.

Reply:
@castrojo Circling back to this. @superm1 indicated his patch was not yet applied to Fedora. We spoke about this in email and I confirmed this state on my own AMD Ryzen 7040 Series unit.

I have alerted Fedora to this and Adam has sent it to Justin to see what can be done to get this Mario's patch integrated into the kernel. In the mean time, I am thinking a separate "just" for rtc_cmos.use_acpi_alarm=1 would be worthwhile in the short term. Long term, ideally, Mario's patch would resolve this once Fedora does their thing. 

Just thinking out loud. Mario may have insights here I don't, but in my mind, a workaround now beats a bug report later while Fedora sorts this out. Thoughts?

Reply:
Sounds good.

Reply:
Another thing that should be incorporated is one of the epp hints fixups. My preference would be to include a patched version of power-profiles-daemon which supports setting the epp platform hints. The unmerged patch set applied to 0.13 (which is the current version shipping in fedora 39 and rawhide) would be my preferred option i.e this  - https://gitlab.freedesktop.org/upower/power-profiles-daemon/-/merge_requests/123 

I've tested this extensively on fc39 and 40 and it seems to be working fine ; will require a new rpm spec applied agianst the 0.13 version and rebuilt into the image.